Changes.md

```diff
@@ -1,3 +1,11 @@
+1.0.x.x (relative to 1.0.4.0)
+=======
+
+Improvements
+------------
+
+Light Editor : Added the ability to select linked objects. It can be accessed by right-clicking a light name or parameter and selecting 'Select Linked Lights'.
```

Comment:
light name ~or parameter~
and selecting 'Select Linked Lights' -> and selecting "Select Linked Objects"

Reply:
Fixed in c4e9998c6288e023356279a85d74ee732d08e27a